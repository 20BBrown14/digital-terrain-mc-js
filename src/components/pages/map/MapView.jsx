import React from 'react';
import './Map.css';

/**
 * Map page view.
 */
const MapView = () => (
  <div>
    <iframe src="https://dynmap-reverse-proxy.herokuapp.com/" className="map-iframe" title="Digital Terrain Dynmap">
      <p>Your browser does not support this type of content.</p>
    </iframe>
  </div>
);

export default MapView;
