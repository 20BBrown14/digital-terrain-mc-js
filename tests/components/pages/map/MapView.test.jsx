import React from 'react';
import MapView from '../../../../src/components/pages/map/MapView';

describe('MapView', () => {
  it('renders a default view', () => {
    const view = render(<MapView />);
    expect(view).toMatchSnapshot();
  });

  it('has an iframe', () => {
    const view = shallow(<MapView />);
    expect(view.find('iframe')).toHaveLength(1);
  });
});
