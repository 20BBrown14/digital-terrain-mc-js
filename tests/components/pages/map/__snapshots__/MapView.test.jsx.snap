// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapView renders a default view 1`] = `
<div>
  <iframe
    class="map-iframe"
    src="http://147.135.107.178:4492/index.html"
    title="Digital Terrain Dynmap"
  >
    &lt;p&gt;Your browser does not support this type of content.&lt;/p&gt;
  </iframe>
</div>
`;
